Makefile

```diff
@@ -8,16 +8,10 @@ FS_MODULES=/usr/lib/freeswitch/mod
 SHELL := /bin/bash
 PROC?=$(shell uname -m)
 
-CC=gcc 
 CFLAGS=-fPIC -O3 -fomit-frame-pointer -fno-exceptions -Wall -std=c99 -pedantic
-ifeq (${PROC},x86_64)
```

Comment:
this may be needed if 32bit code is compiled on x86_64 machines, so I'm not confortable in turning it off.

Reply:
And i never did, sorry for that!
I dont know whats the best way to do this

Reply:
ifeq ($(TARGET_CPU),x86_64)
        CFLAGS+=-m64 -mtune=generic
else ifeq ($(TARGET_CPU),x86)
        CFLAGS+=-m32 -march=i686
endif

Could use sth like this but i dont know if that benefits anyone.

Reply:
Actually, i think this entire thing is not needed and can be dropped.
Reasons:
1) GCC automatically defaults to 32bit if the installed compiler is 32bit (like it is by default on a 32bit machine with an x64 cpu)
2) It only causes problem for other architectures.
3) 32bit linux is going to die soon enough Ubuntu 17.10 already dropped support.
4) I could not find a valid use-case for this code (tried a 32bit vm with an x64 cpu)

I suggest accepting this merge request in its current and original state.
If you can provide me with a solid proof that the code is actually needed i will figure out additional options.

Reply:
ok, granted :) going to merge as is.